sdk/tests/conformance2/textures/misc/tex-storage-compressed-formats.html

```diff
@@ -87,15 +87,28 @@
         var tex = gl.createTexture();
         gl.bindTexture(gl.TEXTURE_2D, tex);
         gl.texStorage2D(gl.TEXTURE_2D, 1, internalformat, 1, 1);
-        wtu.glErrorShouldBe(gl, gl.NO_ERROR, "texStorage2D should succeed for " + enumToString(internalformat));
+        wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+                            "texStorage2D should succeed for " + enumToString(internalformat));
         gl.deleteTexture(tex);
 
-        // Test a 3D texture.
+        // GLES 3.0.4 p147:
+        // "If internalformat is an ETC2/EAC format, CompressedTexImage3D will generate an
+        //  INVALID_OPERATION error if target is not TEXTURE_2D_ARRAY."
```

Comment:
Does CompressedTexImage3D support internalformat which isn't ETC2/EAC foramt? If not, why CompressedTexImage3D  support TEXTURE_3D target? I am a little confused about this.  CompressedTexImage3D supports following compressed internal formats: GL_COMPRESSED_R11_EAC, GL_COMPRESSED_SIGNED_R11_EAC, GL_COMPRESSED_RG11_EAC, GL_COMPRESSED_SIGNED_RG11_EAC, GL_COMPRESSED_RGB8_ETC2, GL_COMPRESSED_SRGB8_ETC2, GL_COMPRESSED_RGB8_PUNCHTHROUGH_ALPHA1_ETC2, GL_COMPRESSED_SRGB8_PUNCHTHROUGH_ALPHA1_ETC2, GL_COMPRESSED_RGBA8_ETC2_EAC, or GL_COMPRESSED_SRGB8_ALPHA8_ETC2_EAC. All of them are ETC2/EAC format. Please refer https://www.khronos.org/opengles/sdk/docs/man3/html/glCompressedTexImage3D.xhtml.